lib/fork.js

```diff
@@ -9,19 +9,15 @@ if (fs.realpathSync(__filename) !== __filename) {
 	console.warn('WARNING: `npm link ava` and the `--preserve-symlink` flag are incompatible. We have detected that AVA is linked via `npm link`, and that you are using either an early version of Node 6, or the `--preserve-symlink` flag. This breaks AVA. You should upgrade to Node 6.2.0+, avoid the `--preserve-symlink` flag, or avoid using `npm link ava`.');
 }
 
-const env = {NODE_ENV: 'test', ...process.env};
-
-// Ensure NODE_PATH paths are absolute
-if (env.NODE_PATH) {
-	env.NODE_PATH = env.NODE_PATH
-		.split(path.delimiter)
-		.map(x => path.resolve(x))
-		.join(path.delimiter);
-}
-
 // In case the test file imports a different AVA install,
 // the presence of this variable allows it to require this one instead
-env.AVA_PATH = path.resolve(__dirname, '..');
+const AVA_PATH = path.resolve(__dirname, '..');
+
+// Ensure NODE_PATH paths are absolute
+const NODE_PATH = process.env.NODE_PATH && process.env.NODE_PATH
```

Comment:
This might be better with a `let NODE_PATH` and `if (process.env.NODE_PATH)`. Currently this will resolve to `'false'`.